vitess-mixin/README.md

```diff
@@ -13,9 +13,13 @@ A set of Grafana dashboards, Prometheus rules and alerts for Vitess, packaged to
         export PATH="$GOPATH/bin:/usr/local/go/bin:$PATH"
         ```
 
-1. Install the go tools: `make tools`, `jb`, `jsonnet`, `jsonnetfmt`, and `promtool` should now be in `$GOPATH/bin`.
+2. Install the go tools: `make tools`, `jb`, `jsonnet`, `jsonnetfmt`, and `promtool` should now be in `$GOPATH/bin`:  
+  - `go install -a github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@latest`  
+  - `go install -a github.com/google/go-jsonnet/cmd/jsonnet@latest`  
+  - `go install -a github.com/google/go-jsonnet/cmd/jsonnetfmt@latest`  
+  - `go install -a github.com/prometheus/prometheus/cmd/promtool`  
```

Comment:
Do you mind making this two separate steps? It is not clear what should be in `$GOPATH/bin`. Let's make a first step where we run `make tools`, and a second step where we do all the `go install`s

Reply:
@frouioui , actually I wasn't sure about `make tools` as well, since it was part of (probably) other toolchain already installed on my machine before the manipulation, so I just didn't touch it.
My point was about making things a bit more explicit for people who aren't professional with Golang tools.
Still, it's not a problem, I will try to repeat this on a clean VM, so I could separate the steps in the README later.